index.html

```diff
@@ -7,21 +7,61 @@
     <title>Awesome Book</title>
     <link rel="stylesheet" href="https://bootswatch.com/5/sketchy/bootstrap.min.css">
     <link rel="stylesheet" href="./style.css">
-    <script defer src="./script.js"></script>
 </head>
 <body>
+    <header>
+        <nav class="navbar navbar-expand navbar-light bg-light">
+            <div class="container-fluid">
+              <a class="navbar-brand" href="#">Awesome Books</a>
+              <div class="collapse navbar-collapse" id="navbarColor03">
+                <ul class="navbar-nav me-auto nav-ul">
+                  <li class="nav-item">
+                    <a class="nav-link list-nav" href="#">List
+                      <span class="visually-hidden">(current)</span>
+                    </a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link add-nav" href="#">Add New</a>
+                  </li>
+                  <li class="nav-item">
+                    <a class="nav-link contact-nav" href="#">Contact</a>
+                  </li>
```

Comment:
- [Optional] Please use an indentation of two spaces in all your documents. This is a convention and also [best practice.](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)